server/lib/flow-event.js

```diff
@@ -28,7 +27,7 @@ const FLOW_BEGIN_EVENT_TYPES = /^flow\.[a-z_-]+\.begin$/;
 const FLOW_ID_KEY = config.get('flow_id_key');
 const FLOW_ID_EXPIRY = config.get('flow_id_expiry');
 
-const ENTRYPOINT_PATTERN = /^[\w\.-]+$/;
+const ENTRYPOINT_PATTERN = /^[\w.-]+$/;
```

Comment:
The period doesn't need escaping here because it's inside a character class. Updated to be consistent with the new regex below.